2021-06-04 12:02:21,929 INFO    MainThread:29500 [internal.py:wandb_internal():91] W&B internal server running at pid: 29500, started at: 2021-06-04 12:02:21.922400
2021-06-04 12:02:21,931 INFO    WriterThread:29500 [datastore.py:open_for_write():77] open: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\run-3i45uhfr.wandb
2021-06-04 12:02:21,931 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: check_version
2021-06-04 12:02:21,933 DEBUG   SenderThread:29500 [sender.py:send():176] send: header
2021-06-04 12:02:21,933 DEBUG   SenderThread:29500 [sender.py:send_request():186] send_request: check_version
2021-06-04 12:02:22,097 DEBUG   SenderThread:29500 [sender.py:send():176] send: run
2021-06-04 12:02:22,117 DEBUG   SenderThread:29500 [git.py:repo():33] git repository is invalid
2021-06-04 12:02:22,351 INFO    SenderThread:29500 [dir_watcher.py:__init__():168] watching files in: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files
2021-06-04 12:02:22,352 INFO    SenderThread:29500 [sender.py:_start_run_threads():699] run started: 3i45uhfr with start time 1622800941
2021-06-04 12:02:22,352 DEBUG   SenderThread:29500 [sender.py:send():176] send: summary
2021-06-04 12:02:22,352 INFO    SenderThread:29500 [sender.py:_save_file():821] saving file wandb-summary.json with policy end
2021-06-04 12:02:22,357 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: run_start
2021-06-04 12:02:23,355 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\wandb-summary.json
2021-06-04 12:02:24,309 DEBUG   HandlerThread:29500 [meta.py:__init__():39] meta init
2021-06-04 12:02:24,309 DEBUG   HandlerThread:29500 [meta.py:__init__():53] meta init done
2021-06-04 12:02:24,309 DEBUG   HandlerThread:29500 [meta.py:probe():210] probe
2021-06-04 12:02:24,317 DEBUG   HandlerThread:29500 [git.py:repo():33] git repository is invalid
2021-06-04 12:02:24,317 DEBUG   HandlerThread:29500 [meta.py:_save_pip():57] save pip
2021-06-04 12:02:24,318 DEBUG   HandlerThread:29500 [meta.py:_save_pip():71] save pip done
2021-06-04 12:02:24,318 DEBUG   HandlerThread:29500 [meta.py:_save_conda():78] save conda
2021-06-04 12:02:24,358 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\requirements.txt
2021-06-04 12:02:24,358 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\conda-environment.yaml
2021-06-04 12:02:27,124 DEBUG   HandlerThread:29500 [meta.py:_save_conda():86] save conda done
2021-06-04 12:02:27,124 DEBUG   HandlerThread:29500 [meta.py:probe():252] probe done
2021-06-04 12:02:27,127 DEBUG   SenderThread:29500 [sender.py:send():176] send: files
2021-06-04 12:02:27,127 INFO    SenderThread:29500 [sender.py:_save_file():821] saving file wandb-metadata.json with policy now
2021-06-04 12:02:27,139 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: stop_status
2021-06-04 12:02:27,140 DEBUG   SenderThread:29500 [sender.py:send_request():186] send_request: stop_status
2021-06-04 12:02:27,359 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_modified():229] file/dir modified: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\conda-environment.yaml
2021-06-04 12:02:27,359 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\wandb-metadata.json
2021-06-04 12:02:27,360 INFO    Thread-12 :29500 [dir_watcher.py:_on_file_created():216] file/dir created: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\output.log
2021-06-04 12:02:27,714 INFO    Thread-15 :29500 [upload_job.py:push():133] Uploaded file C:\Users\ROBERT~1\AppData\Local\Temp\tmphtz7h_9nwandb\34po7bh0-wandb-metadata.json
2021-06-04 12:02:42,342 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: stop_status
2021-06-04 12:02:42,342 DEBUG   SenderThread:29500 [sender.py:send_request():186] send_request: stop_status
2021-06-04 12:02:52,933 DEBUG   SenderThread:29500 [sender.py:send():176] send: stats
2021-06-04 12:02:57,551 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: stop_status
2021-06-04 12:02:57,551 DEBUG   SenderThread:29500 [sender.py:send_request():186] send_request: stop_status
2021-06-04 12:03:12,753 DEBUG   HandlerThread:29500 [handler.py:handle_request():121] handle_request: stop_status
2021-06-04 12:03:12,753 DEBUG   SenderThread:29500 [sender.py:send_request():186] send_request: stop_status
2021-06-04 12:03:23,566 DEBUG   SenderThread:29500 [sender.py:send():176] send: stats
2021-06-04 12:03:26,975 WARNING MainThread:29500 [internal.py:is_dead():367] Internal process exiting, parent pid 27596 disappeared
2021-06-04 12:03:26,975 ERROR   MainThread:29500 [internal.py:wandb_internal():143] Internal process shutdown.
2021-06-04 12:03:27,568 INFO    WriterThread:29500 [datastore.py:close():258] close: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\run-3i45uhfr.wandb
2021-06-04 12:03:27,568 INFO    SenderThread:29500 [sender.py:finish():925] shutting down sender
2021-06-04 12:03:27,568 INFO    HandlerThread:29500 [handler.py:finish():632] shutting down handler
2021-06-04 12:03:27,568 INFO    SenderThread:29500 [dir_watcher.py:finish():282] shutting down directory watcher
2021-06-04 12:03:28,397 INFO    SenderThread:29500 [dir_watcher.py:finish():312] scan: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files
2021-06-04 12:03:28,397 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\conda-environment.yaml conda-environment.yaml
2021-06-04 12:03:28,397 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\config.yaml config.yaml
2021-06-04 12:03:28,398 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\output.log output.log
2021-06-04 12:03:28,401 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\requirements.txt requirements.txt
2021-06-04 12:03:28,404 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\wandb-metadata.json wandb-metadata.json
2021-06-04 12:03:28,404 INFO    SenderThread:29500 [dir_watcher.py:finish():318] scan save: C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\wandb-summary.json wandb-summary.json
2021-06-04 12:03:28,405 INFO    SenderThread:29500 [file_pusher.py:finish():176] shutting down file pusher
2021-06-04 12:03:28,405 INFO    SenderThread:29500 [file_pusher.py:join():181] waiting for file pusher
2021-06-04 12:03:29,000 INFO    Thread-18 :29500 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\requirements.txt
2021-06-04 12:03:29,014 INFO    Thread-19 :29500 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\wandb-summary.json
2021-06-04 12:03:29,019 INFO    Thread-17 :29500 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\config.yaml
2021-06-04 12:03:29,028 INFO    Thread-16 :29500 [upload_job.py:push():133] Uploaded file C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_120220-3i45uhfr\files\conda-environment.yaml
2021-06-04 12:03:29,719 INFO    MainThread:29500 [internal.py:handle_exit():78] Internal process exited
